src/FSharp.AWS.DynamoDB/Diagnostics.fs

```diff
@@ -0,0 +1,70 @@
+namespace FSharp.AWS.DynamoDB
+
+module Diagnostics =
```

Comment:
see also https://github.com/jet/equinox/blob/master/src/Equinox/Tracing.fs https://github.com/jet/equinox/blob/master/src/Equinox.MessageDb/Tracing.fs for other helper layout ideas